contracts/NFTStorefront.cdc

```diff
@@ -122,36 +130,58 @@ pub contract NFTStorefront {
         pub var purchased: Bool
         // The Type of the NonFungibleToken.NFT that is being listed.
         pub let nftType: Type
-        // The ID of the NFT within that type.
+        // The Resource ID of the NFT which can only be set in the contract
+        //TODO how can this better be enforced?
+        pub let nftUUID: UInt64
```

Comment:
still unclear what this `nftUUID` is

Reply:
https://docs.onflow.org/cadence/language/resources/#resource-identifier